test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
we can remove this file. In development mode, we don't create/push the Rspec files.